Sources/NukeUI/FetchImage.swift

```diff
@@ -184,6 +188,10 @@ public final class FetchImage: ObservableObject, Identifiable {
                 handle(result: .failure(error))
             }
         }
+
+        if let task = imageTask {
```

Comment:
The task is always going to be empty at this point, to it's not going to get called. I'm not entire sure why I added this `load()` variant in the first place, to be honest, but I think it would be acceptable to not call `onStart` here.

Reply:
Yup, I agree, I wasn't sure if you had something planned so I added it just in case. I'll remove it